### PR TITLE
Good bye jbuilder 👋

### PR DESCRIPTION
**Pull request by ghost**
This PR removes the `jbuilder` binary as well as the `wp` library. It also removes the conflicts between the `dune` and `jbuilder` packages. Following this, the `jbuilder` and `dune` package should be co-installable.

### CODE REVIEW

**Review by rgrinberg — APPROVED** [review]
Looks good. Let's make sure the change log stays up to date though

**Comment by diml** [comment]
Ah yes, good point. I'll update it as soon as Travis is done.

File: dune.opam [hunk 1]
```diff
@@ -32,9 +32,9 @@ depends: [
   "base-threads"
 ]
 conflicts: [
-  "jbuilder" {!= "transition"}
   "odoc" {< "1.3.0"}
   "dune-release" {< "1.3.0"}
+  "jbuilder" {= "transition"}
```
Comment (dra27) [inline dune.opam]: Doesn’t this want to be just `jbuilder`? Dune 2 conflicts with *all* versions, right?

Comment (diml) [reply dune.opam]: Nope. Since it doesn't install a `jbuilder` binary, it can be installed alongside the original jbuilder :)

**Pull request merged**